tests/hikari/test_audit_logs.py

```diff
@@ -57,6 +102,42 @@ def test_repr(self):
         assert repr(audit_log) == "UnrecognisedAuditLogEntryInfo(test='test2', test3=98)"
 
 
+class TestAuditLogEntry:
+    @pytest.mark.asyncio
+    async def test_fetch_user_when_no_user(self):
+        model = audit_logs.AuditLogEntry(
+            app=mock.AsyncMock(),
+            id=123,
+            target_id=None,
+            changes=[],
+            user_id=None,
+            action_type=0,
+            options=None,
+            reason=None,
+        )
+
+        assert await model.fetch_user() is None
```

Comment:
btw is this the decided way for handling this scenario? Kind of feels similar to the webhook methods where if the token isn't present we raise as we can't finish the operation without it and not sure if there's any semantic difference that i'm missing

Reply:
Will have a think

Reply:
Yea, this is the way we are going to do to it